private/buf/cmd/buf/command/beta/registry/draft/draftdelete/draftdelete.go

```diff
@@ -36,8 +36,8 @@ func NewCommand(
 ) *appcmd.Command {
 	flags := newFlags()
 	return &appcmd.Command{
-		Use:   name + " <buf.build/owner/repo:draft>",
-		Short: "Delete a draft of a BSR repository by name.",
+		Use:   name + " <buf.build/owner/name:draft>",
```

Comment:
(Know this is a simple find and replace but for clarity) is there a distinction between a `BSR module` versus a `module`?

Reply:
technically there are local modules which are just the module before it's pushed to the bsr